packages/coinstac-graphql/src/data/computation-fields.js

```diff
@@ -0,0 +1,14 @@
+const computationFields = {
```

Comment:
Before I added the JSON type, I had about six of these strings that were used in multiple places in `schema.js` and it was easier to import a single object containing multiple strings rather than importing the strings individually. However, once we found the JSON type, those other functions were unnecessary and I removed them. I didn't revert to a single string as I thought we might add more as `schema.js` grows. I may be wrong about that, but I didn't see the harm in leaving it in there for now.